packages/fxa-auth-server/lib/sentry.js

```diff
@@ -246,6 +246,23 @@ async function configureSentry(server, config, processName = 'key_server') {
         reportSentryError(event.error, request);
       }
     });
+
+    /**
+     * This captures joi response validation errors
+     * They will escape sentry capture otherwise
+     */
+    server.ext('onPreResponse', (request, h) => {
+      const response = request.response;
+      if (
+        response.isBoom &&
+        response.output.statusCode >= 500 &&
```

Comment:
Is it possible that output could be undefined? I looked at the hapi docs briefly and nothing popped out at me.

Reply:
so this should short circuit since if isBoom is false, the whole conditional fails. if isBoom is true then output is defined according to the hapi docs. i could (and will) also check add a check though for output not being undefined. i just need to figure out whats going on with the test failures first.